src/main/java/com/uber/cadence/worker/Worker.java

```diff
@@ -480,6 +544,11 @@ public void start() {
       }
     }
 
+    /**
+     * Shuts down Poller used for sticky workflows and all workers created by this factory.
+     *
+     * @param timeout
```

Comment:
Document what happens on timeout. Does it return or throw?

Reply:
resolved